apps/fastapi_app.py

```diff
@@ -10,11 +10,10 @@
 app = FastAPI()
 app.include_router(vulnerable_routes)
 
-# TODO: PYT-1701
-# if os.environ.get("VULNPY_USE_CONTRAST"):
-#     from contrast.fastapi import ContrastMiddleware
-#
-#     app.middleware("http")(ContrastMiddleware(app))
+if os.environ.get("VULNPY_USE_CONTRAST"):
+    from contrast.fastapi import ContrastMiddleware
+
+    app.middleware("http")(ContrastMiddleware(app))
```

Comment:
is it possible to install the middleware like this?
```
app.add_middleware(ContrastMiddleware)
```

Reply:
great question. I tried and it didn't work but now I'm smarter so I think I can try again...

Reply:
given the docs you'd think so? https://fastapi.tiangolo.com/advanced/middleware/

Reply:
yeah, my concern is that if it's not working like the example in the docs it's an indication that something's wrong on our end

Reply:
Ok looks like our options are
`app.middleware("http")(ContrastMiddleware(app))` as I have it

`app.add_middleware(ContrastMiddleware, app=app)`
but I have some ideas to make it
``app.add_middleware(ContrastMiddleware)`

hang tight

`app.add_middleware(UnicornMiddleware, some_config="rainbow")`